stories/shared-components/species-lozenge/SpeciesLozenge.stories.tsx

```diff
@@ -15,9 +15,9 @@
  */
 
 import SpeciesLozenge from 'src/shared/components/selected-species/SpeciesLozenge';
+import WrapInRedux from './SpeciesLozengeReduxWrapper';
```

Comment:
Curious about whether this a better alternative to react hooks?

Reply:
I am not sure I understand... What alternative do you have in mind?